### PR TITLE
test fxEqnAnnualSine

### LINKED ISSUE #72: fcEqnAnnualSine  - develop tests
as per title

### PR DESCRIPTION
**Pull request by j-emberton**
closes #72 

this function takes a description of a sine wave (v offset, amplitude and h offset), and a time array in days of the year and returns a time series of that sine wave at those specified days.

All tests run.

### CODE REVIEW

**Review by isaacaka — APPROVED** [review]
Looks good

File: tests/unit_tests/test_ustar_cp/test_fcEqnAnnualSine.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+        # Case 3: Sine Wave with Offset
+        ([2, 1, 0], [0, 90, 180, 270, 360], [2, 2.9997458409224182, 2.0450774993382357, 1.0022866566018664, 1.909936644521285]),
+
+        # Case 4: Phase Shift
+        ([0, 1, 90], [0, 90, 180, 270, 360], [-0.9997458409224182, 0, 0.9997458409224182, 0.04507749933823567, -0.9977133433981337]),
+
+        # Case 5: Negative Amplitude
+        ([0, -1, 0], [0, 90, 180, 270, 360], [0, -0.9997458409224182, -0.04507749933823567, 0.9977133433981337, 0.09006335547871482]),
+
```
Comment (dorchard) [inline tests/unit_tests/test_ustar_cp/test_fcEqnAnnualSine.py:24]: Could we add a little variation, e.g., with smaller time frames and non monotonic time frames?
```suggestion
        # Case 6: Pure Sine Wave, Zero Offset, small time frame
        ([0, 1, 0], [0, 180, 360], [0, 0.04507749933823567, -0.09006335547871482]),

        # Case 7: Pure Sine Wave, Zero Offset, non monotonic time frame
        ([0, 1, 0], [0, 360, 180], [0,  -0.09006335547871482, 0.04507749933823567]),
```

**Review by dorchard — COMMENTED** [review]
Lgtm apart from adding a couple more tests which I've added in suggestion mode (checked them already).

**Comment by j-emberton** [comment]
good suggestion Dom - committed

**Review by dorchard — APPROVED** [review]

**Comment by dorchard** [comment]
Okay I'm happy for this to be merged now.

**Pull request merged**